rollup/ccc/logger.go

```diff
@@ -143,6 +154,15 @@ func (l *Logger) logSha256(inputLen uint64) {
 	l.sha256Usage += numBlocks * blockRows
 }
 
+// LogKeccakChunks records the number of chunks used for keccak given the length of bytes.
+func (l *Logger) LogKeccakChunks(length uint64) {
+	l.keccakChunkCount += computeKeccakChunks(length)
+}
+
+func (l *Logger) LogKeccakAllL2TxsLen(length uint64) {
+	l.keccakAllL2TxsLen += length
+}
+
```

Comment:
We can add txn size to `TxnContext` (similar to `IsL1MessageTx`) and not need these helper functions at all. This will solve the issue I described in the previous comment as well.